src/editors/config/URL.tsx

```diff
@@ -22,6 +22,29 @@ export const URLEditor = (props: DataSourcePluginOptionsEditorProps<InfinityOpti
         onChange={(e) => setUrl(e.currentTarget.value || '')}
         onBlur={onURLChange}
       />
-    </div>
+    </Stack>
+  );
+};
+
+export const URLMiscEditor = (props: DataSourcePluginOptionsEditorProps<InfinityOptions>) => {
+  const { options, onOptionsChange } = props;
+  const { jsonData = {} } = options;
+  return (
+    <Stack direction={'column'}>
+      <Stack>
+        <InlineLabel
+          width={36}
+          tooltip={`By default infinity only allow GET/POST HTTP methods to reduce the risk of destructive/accidental payloads. If you need non GET/POST methods, make use of this setting with caution. Note: Infinity don't evaluate any permissions against the underlying API`}
+        >
+          Allow non GET / POST HTTP verbs
```

Comment:
```suggestion
          Allow destructive HTTP methods
```

Same reasoning as above.